index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
             </li>             
      </ul>               
     </section>
+    <section class="works">
+        <div class="title">
+           <h2>My Recent Works</h2>
+           <div class="line"></div>
+        </div>
+        <div class="sample">
+           <img src="./assets/images/works-section.png" alt="works">
+           <h3>Multi-Post Stories</h3>
+           <p class="works-para">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="tags">
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">HTML</li>
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
+           <button type="button">See Project</button>
+        </div>
+        <div class="works-grid">
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
```

Comment:
- [ ] Kindly use the correct font sizes for the ```<p>``` as provided by the Figma design. Currently, the 1.1rem used converted to px makes around 17.6px. Kindly use 16px as the provided font size for this section

**This is how I am seeing your project:**

![image](https://user-images.githubusercontent.com/86318284/185756715-24d79aff-2319-45ae-a747-3a8b3259d89e.png)


**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185756729-3a858815-3592-4d5c-a324-50892fb08df9.png)

**Kindly implement this in all your projects paragraph tags**